packages/web/src/components/drawer/Drawer.tsx

```diff
@@ -61,6 +62,12 @@ export type DrawerProps = {
   isFullscreen?: boolean
 }
 
+const getHeight = (contentRef: RefObject<HTMLDivElement>) => {
+  if (!contentRef.current) return 0
+
+  return contentRef.current.getBoundingClientRect().height
```

Comment:
I don't know Spring very well. Probably worth confirming that this doesn't run as part of callbacks or anything like that, since it triggers a layout in order to computer the client rect.
I would _hope_ / assume that your callback to `useSpring` below gets executed inside a `requestAnimationFrame`, though.